message/html/compose.js

```diff
@@ -77,9 +83,10 @@ exports.create = function (api) {
       textArea.value = textArea.value.slice(0, pos) + insertLink + textArea.value.slice(pos)
 
       console.log('added:', file)
-    })
+    }, opts)
```

Comment:
Haven't looked but is that an existing core feature or are you suggesting it should exist.
Sounds like a great idea